### ISSUE TITLE
new term request "copepod prosome"

### ISSUE THREAD

**Issue by diatomsRcool** [obophenotype/uberon#1395]
Please add as a child to "tagma" http://purl.obolibrary.org/obo/UBERON_6000002
"copepod prosome"
The forward section of the copepod body containing the head. It is separated form the rear section of the body (urosome) by a single point of articulation.

**Comment by diatomsRcool** [comment]
https://en.wikipedia.org/wiki/Tagma_(biology) for reference

**Comment by cmungall** [comment]
cc @dosumis 

@diatomsRcool - do you want access to the insect module?
 You should be able to go ahead and make pull requests here https://github.com/obophenotype/insect-anatomy-ontology

**Comment by cmungall** [comment]
doh copepods aren't insects... but nevertheless the insect module has a lot of stuff useful for arthrorpods, thoughts @dosumis ?

**Comment by diatomsRcool** [comment]
yeah, i guess it just depends if non-insect parts are welcome in the insect anatomy ontology @dosumis ?

**Comment by dosumis** [comment]
We sold probably add some high level arthropod terms to Uberon.  For this
term we could add tagma to Uberon as a super class for prosome.

**Comment by diatomsRcool** [comment]
tagma is in Uberon, I think

**Comment by diatomsRcool** [comment]
So where did we land with this? Am I clear to add to the insect module?

**Comment by dosumis** [comment]
Sorry to be slow on this. 

As everything in the insect anatomy has an Uberon ID I guess the only issues are:
- Would it be confusing for prosome to live here (who would think to look?)
- Will there be ID range issues if we put the term in the insect module and try to move it out later ? (I don't see a range specified here, but presumably we need some way of preventing stomping on UBERON iris when parts of Uberon are maintained in different files (@cmungall  please advise - I should probably know how this works).

**Comment by cmungall** [comment]
> who would think to look

Note that the insect module actually gets incorporated into uberon in the main release. So from a user perspective everything is equally easy (or hard) to find. the module split is really about making it easier for a community to edit part of the ontology.

ID ranges are currently 6nnnnnn (see https://github.com/obophenotype/insect-anatomy-ontology/issues/7)

My preference here is simply to rename https://github.com/obophenotype/insect-anatomy-ontology to https://github.com/obophenotype/arthropod-anatomy-ontology, and to allow classes like [UBERON:6000002](http://purl.obolibrary.org/obo/UBERON_6000002) ! tagma to be used for arthropods.

**Comment by dosumis** [comment]
> My preference here is simply to rename https://github.com/obophenotype/insect-anatomy-ontology to https://github.com/obophenotype/arthropod-anatomy-ontology, and to allow classes like UBERON:6000002 ! tagma to be used for arthropods.

OK.  Happy with that if you are.  Tagma definition should be updated as its currently very insect specific.  

Dug this proposed tagma def out from old work on AAO that never saw the light of day (would be great to find time/resources/people to revisit this work):

def: "A subdivision of the organism along the anterior-posterior axis that consists of multiple adjacent segments that are either fused, and/or share more structure with each other than with segments in adjacent tagmata."

**Comment by diatomsRcool** [comment]
Sure.

**Comment by diatomsRcool** [comment]
Just so I don't screw this up......
I'm 1. changing the name of "insect-anatomy-ontology" to "arthropod-anatomy-ontology", 2. editing the definition of tagma in uberon, 3. adding copepod prosome to arthropod-anatomy-ontology.
Correct?

**Comment by dosumis** [comment]
> I'm 1. changing the name of "insect-anatomy-ontology" to "arthropod-anatomy-ontology", 2. editing the definition of tagma in uberon, 3. adding copepod prosome to arthropod-anatomy-ontology.

> Correct? 

Yep.  But looks like tagma  (UBERON:6000002) is currently in the 'insect-anatomy-ontology' file (everything there has an Uberon ID).  Also make sure to set ID range appropriately.

**Comment by diatomsRcool** [comment]
I can't seem to get my edited obo file to save. Should I not be doing this in Protege?

**Comment by cmungall** [comment]
You can edit obo in protege. We like to keep source in obo for diff purposes.

In the absence of editor manual for the insect/arthro module, the GO docs should be a good proxy: http://go-ontology.readthedocs.io/en/latest/

**Comment by diatomsRcool** [comment]
I think this might be what's happening
https://github.com/protegeproject/protege/issues/501

**Comment by cmungall** [comment]
does it look like this?

![image](https://user-images.githubusercontent.com/50745/35072209-798f4674-fb98-11e7-8a7a-a53c69990cb1.png)

**Comment by cmungall** [comment]
I know what this is.. hold on..

**Comment by diatomsRcool** [comment]
Yes, it looks like this
<img width="949" alt="screen shot 2018-01-22 at 4 30 52 pm" src="https://user-images.githubusercontent.com/5140552/35245576-c74d41b6-ff91-11e7-8165-ba16a267d518.png">

**Comment by diatomsRcool** [comment]
Where are we with this?

**Comment by cmungall** [comment]
Do you get this with latest Protege beta?

**Comment by diatomsRcool** [comment]
I'm using 5.2.0
Is that still beta?
